lib/util/http/azure.ts

```diff
@@ -0,0 +1,30 @@
+import type { AzureBodyPaginated } from '../../types/platform/azure';
+import type { HttpOptions, HttpResponse } from './types';
+import { Http } from '.';
+
+export class AzureHttp extends Http<HttpOptions> {
+  http: any;
+  constructor(type = 'azure', options?: HttpOptions) {
+    super(type, options);
+  }
+
+  async getJsonPaginated<T>(
```

Comment:
Don't use new function for pagiantion. Checkout github and gitab http for pagination.